File: python/prism_sdk/local_evolution_integrity_inference.py
"""Mutation P32 local inference evolution-integrity feature F01."""
from .evolution_integrity_support import EvolutionRequest4,EvolutionCard7,EvolutionIntegrityError,manifest,qualify
FEATURE_ID="AFA-mutation-P32-F01";CONTRACT_VERSION="mutation-local_evolution_integrity_inference/1.0"
def local_evolution_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="inference")
def qualify_local_evolution_integrity_inference(request:EvolutionRequest4)->EvolutionCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="inference")

File: crates/mutation/src/local_evolution_integrity_inference.rs
//! Mutation P32 local inference evolution-integrity feature.
use super::evolution_integrity_support::{manifest,qualify,EvolutionCard7,EvolutionIntegrityError,EvolutionRequest4};
pub const FEATURE_ID:&str="AFA-mutation-P32-F01";pub const CONTRACT_VERSION:&str="mutation-local_evolution_integrity_inference/1.0";
pub fn local_evolution_integrity_inference_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"local","inference")}
pub fn qualify_local_evolution_integrity_inference(request:&EvolutionRequest4)->Result<EvolutionCard7,EvolutionIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"local","inference")}
